File: Python/1051 - Imposto de Renda.py
renda = float(input())

imposto = 0.0

if renda <= 2000.00:
    print("Isento")
else:
    if renda > 2000.00:
        valor = min(renda, 3000.00) - 2000.00
        imposto += valor * 0.08
    if renda > 3000.00:
        valor = min(renda, 4500.00) - 3000.00
        imposto += valor * 0.18
    if renda > 4500.00:
        valor = renda - 4500.00
        imposto += valor * 0.28

    print(f"R$ {imposto:.2f}")

File: Clojure/1051 - Imposto de Renda.clj
(let [renda (Double/parseDouble (read-line))
      imposto (atom 0.0)]
  (if (<= renda 2000.0)
    (println "Isento")
    (do
      (when (> renda 2000.0)
        (let [valor (- (min renda 3000.0) 2000.0)]
          (swap! imposto + (* valor 0.08))))
      (when (> renda 3000.0)
        (let [valor (- (min renda 4500.0) 3000.0)]
          (swap! imposto + (* valor 0.18))))
      (when (> renda 4500.0)
        (let [valor (- renda 4500.0)]
          (swap! imposto + (* valor 0.28))))
      (println (format "R$ %.2f" @imposto)))))